assets/src/css/admin/sale-banners.scss

```diff
@@ -11,12 +11,6 @@
     row-gap: 1.875rem;
     font-family: 'Open Sans', system-ui, sans-serif;
 
-    /* Settings page specific styles */
-    .give-settings-page & {
-        margin-top: 2rem;
-        margin-bottom: 3.25rem;
-    }
-
     #give-in-plugin-upsells & {
```

Comment:
Was this css change intentional?

Reply:
No. I meant to put it on `develop`, which I did cherry-pick later. So just ignore this.